src/components/DimensionsPanel/DndDimensionItem.js

```diff
@@ -41,11 +42,11 @@ export class DndDimensionItem extends Component {
                             innerRef={provided.innerRef}
                             {...provided.draggableProps}
                             {...provided.dragHandleProps}
-                            className={
-                                snapshot.isDragging
-                                    ? styles.dragging
-                                    : styles.notDragging
-                            }
+                            className={cx({
+                                [styles.dragging]: snapshot.isDragging,
+                                [styles.notDragging]: !snapshot.isDragging,
+                                [styles.assignedCategories]: id === 'co',
```

Comment:
Should "co" be put into a constant?

Reply:
Yeah, ideally it should. I looked for a suitable file, but couldn't fine one. Then I considered if it was worth creating a completely new file for a single 2-character constant that is only used in 2 places, and concluded it probably isn't.

But I'd be happy to move it somewhere else. Could you advise which file to use/create?

Reply:
Import `DIMENSION_ID_ASSIGNED_CATEGORIES` from `@dhis2/analytics`

Reply:
🙏